storybook/src/mock/documents.mock.js

```diff
@@ -0,0 +1,80 @@
+import image from './nuxeo-logo.jpg';
+
+const DOCUMENT1 = {
+  'entity-type': 'document',
+  uid: '80837e7e-fbfe-4f4f-8d7e-226b285cec29',
+  facets: [],
+  properties: {
+    'file:content': {
+      name: 'stuff.json',
+      data: '',
+    },
+  },
+  contextParameters: {
+    permissions: ['Write'],
+  },
+};
+
+const DOCUMENT2 = {
+  'entity-type': 'document',
+  uid: '71b4cf9c-ae79-4a80-b1e5-85f5684f15c3',
+  facets: [],
+  properties: {
+    'file:content': {
+      name: 'stuff.json',
+      data: '',
+    },
+  },
+  contextParameters: {
+    favorites: {
+      isFavorite: false,
+    },
+    permissions: ['Write'],
+  },
+};
+
+const DOCUMENT3 = {
+  'entity-type': 'document',
+  uid: '15067572-ccb1-4cad-8e40-4911f626d56c',
+  facets: [],
+  properties: {
+    'file:content': {
+      name: 'stuff.json',
+      data: '',
+    },
+  },
+  contextParameters: {
+    favorites: {
+      isFavorite: false,
+    },
+    permissions: ['Write'],
+  },
+};
+
+export const DOCUMENT_DOWNLOAD = {
+  'entity-type': 'document',
+  uid: '99c7488a-8eae-4512-bac9-4f7fe60b9e44',
+  facets: [],
+  properties: {
+    'file:content': {
+      data: image,
```

Comment:
Could we just include this on one of the other documents and just reuse it, or do we need to have this 4th document on top of the other 3?

Reply:
After some investigation, we saw that other storybook projects were facing the same problem (repeating dummy data). There are two main ways other projects deal with this situation:
1. creating inline dummy content inside the story;
2. creating a separated file, with inline content as well.

This happens because there's not a common use case where a helper function to generate documents would be useful.

However, since we need three documents with different `uid`'s in order to `nuxeo-move-documents-up-button` and `nuxeo-move-documents-down-button` work, I've created a little helper to generate those documents. This way we save some lines of code.